lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -54,14 +70,16 @@ def ask_for_repmgrd_configuration
 
     def activate
       say("Configuring Replication Standby Server...")
-      data_dir_empty? &&
+      initialize_postgresql_disk if disk
+      PostgresAdmin.prep_data_directory &&
+        data_dir_empty?
```

Comment:
Then we will raise if `initialize_postgresql_disk` fails so we will not attempt to execute the later tasks and `PostgresAdmin.prep_data_directory` is not expected to fail as most of the commands are `rm -rf`.